x-pack/legacy/plugins/siem/public/containers/hosts/first_last_seen/index.ts

```diff
@@ -35,10 +35,10 @@ export function useFirstLastSeenHostQuery<TCache = object>(
   sourceId: string,
   apolloClient: ApolloClient<TCache>
 ) {
-  const [loading, updateLoading] = useState(false);
-  const [firstSeen, updateFirstSeen] = useState(null);
-  const [lastSeen, updateLastSeen] = useState(null);
-  const [errorMessage, updateErrorMessage] = useState(null);
+  const [loading, updateLoading] = useState<boolean>(false);
+  const [firstSeen, updateFirstSeen] = useState<Date | null>(null);
+  const [lastSeen, updateLastSeen] = useState<Date | null>(null);
```

Comment:
Here where it can be a Date or null it makes sense to use a template as otherwise it will infer it to be only a null.
👍